frontend/src/components/Forms/incidents/AddIncidentFollowUpForm.js

```diff
@@ -0,0 +1,117 @@
+// <label> is being used as is to replicate ant design structure of other rendered fields but,
+// this causes a linting error. Disabling this rule for this file as jsx structure does not allow
+// disabling it on the specific line.
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
generated, or is this us?

Reply:
Looks generated, its for the Recomendations component. @nayr974 ?